src/openqaoa-core/algorithms/workflow_properties.py

```diff
@@ -262,6 +261,62 @@ def __init__(
     #     self._cvar_alpha = value
 
 
+class ErrorMitigationProperties(WorkflowProperties):
+    """
+    Optional, choose an error mitigation technique for the QAOA circuit. Currently supports only SPAM Twirling.
+
+    Parameters
+    ----------
+    error_mitigation_technique: str
+        The name of the error mitigation technique. Only 'spam_twirling' supported for now.
+    n_batches: Optional[int] = int
+        Number of batches into the total number of shots is divided into.
+    calibration_data_location: str
+        The path to the file containing calibration data for the specific device.
+    """
+
+    def __init__(
+        self,
+        error_mitigation_technique = None,
+        n_batches = 10,
+        calibration_data_location = None,
+    ):
+        self.error_mitigation_technique = error_mitigation_technique
```

Comment:
Always cast the user input to lower string. Otherwise, the code Breaks if I say `error_mitigation_technique='SPAM_twirling'`

Reply:
Done. Please review

Reply:
Sorry, I don't see it!

I'd change the code to

```
    def __init__(
        self,
        error_mitigation_technique: str = None,
        n_batches:int = 10,
        calibration_data_location:Optional[str] = None,
    ):
        self.error_mitigation_technique = error_mitigation_technique.lower()
```

Reply:
okay, changed, thanks